packages/core/package.json

```diff
@@ -43,9 +43,14 @@
   },
   "devDependencies": {
     "@backstage/cli": "^0.1.1-alpha.1",
+    "@backstage/test-utils": "^0.1.1-alpha.1",
+    "@backstage/theme": "^0.1.1-alpha.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "react-router": "^5.1.2"
+  },
+  "peerDependencies": {
+    "@backstage/theme": "^0.1.1-alpha.1"
```

Comment:
we'll likely move away from peer deps, since it doesn't work well with workspaces